src/components/Home/LatestArticles/LatestArticles.tsx

```diff
@@ -4,42 +4,88 @@ import { Inner } from '../../Common/Inner';
 import { LatestArticlesQueryEdge, useLatestArticles } from './queries/latest-articles';
 import { GatsbyImage } from 'gatsby-plugin-image';
 import { Link } from 'gatsby';
+import { theme } from '../../../theme';
+import { Section } from '../../Common/Section';
 
-const Outer = styled.div``;
+const ImageContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  border-bottom: 2px solid ${theme.cardBorder};
+`;
+
+const Image = styled(GatsbyImage)`
+  maxwidht: 200px;
+  maxheight: 64px;
```

Comment:
`max-width` and `max-height`
they can be removed as gatsby image component has aspect ratio set.